tensorflow_addons/utils/types.py

```diff
@@ -50,6 +53,7 @@
     tf.Tensor,
     tf.SparseTensor,
     tf.Variable,
+    keras_tensor.KerasTensor,
```

Comment:
This is to fix seq2seq testing. Unfortunetely it's not clear if a fix for https://github.com/tensorflow/tensorflow/issues/44613 will land in TF2.4 so it may be required to add this private import for a bit.

@qlzh727 any issue?